.github/workflows/semantic-release.yml

```diff
@@ -13,10 +13,10 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
+          node-version: '20.x'
       - run: npm ci
       - name: Semantic Release
-        uses: cycjimmy/semantic-release-action@v2
+        uses: codfish/semantic-release-action@v2
```

Comment:
why the switch? Try bumping to `cycjimmy/semantic-release-action@v3`

Reply:
Just saw this, `cycjimmy`'s one was a fork which led to issues while the action was running (I can't recall which ones now other than the fork being out-of-date and the initial version having bugfixes for the issue).